README.md

```diff
@@ -13,8 +13,6 @@ To resolve this, execute the following script:
 ```bash
 git commit -a -m "Saving my work, just in case"
 git branch my-new-branch
-
-git checkout main
```

Comment:
It turns out that `git branch`  does not switch to the new branch, so this line is not needed.